package/src/frontend/browser/safariPlatform.js

```diff
@@ -0,0 +1,6 @@
+import {browser} from './browser';
+import {agent} from './Agent';
+
+browser.feature('safari platform', function() {
+    return agent.matchesMobileSafari() || agent.matchesSafari();
```

Comment:
@tf we probably do not need mobile safari ? since I think editor is not supported for mobile version of safari.
and in preview mode it works fine.
But I could not test it on safari